src/integration-test/resources/GetOrganisationStatusesIT/__files/test2_not_found_in_casemanagement_but_match_in_casestatus_db/expected-response.json

```diff
@@ -1,10 +1,18 @@
 [
 	{
+		"caseType": "Beställa nybyggnadskarta",
+		"externalCaseId": "3447",
+		"firstSubmitted": "2022-02-09 09:51",
 		"lastStatusChange": "2022-02-09 09:51",
 		"openEErrand": true,
+		"status": "Inskickat"
+	},
+	{
+		"caseType": "Beställa nybyggnadskarta",
```

Comment:
Is there some kind of encoding issue here?

Reply:
Everything looks fine on my computer. Github shenanigans?